### PR TITLE
Add 'can-extend-session' key

### PR DESCRIPTION
**Pull request by tfpauly**
Add key for extending sessions. Implements #31.

### CODE REVIEW

File: draft-ietf-capport-api.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ The following keys are defined at the top-level of the JSON structure returned b
 - "captive" (required, boolean): indicates whether the client is in a state of captivity, i.e it has not satisfied the conditions to access the external network. If the client is captive (i.e. captive=true), it can still be allowed enough access for it to perform server authentication {{server-auth}}.
 - "user-portal-url" (optional, string): provides the URL of a web portal with which a user can interact.
 - "venue-info-url" (optional, string): provides the URL of a webpage or site on which the operator of the network has information that it wishes to share with the user (e.g., store info, maps, flight status, or entertainment).
+- "can-extend-session" (optional, boolean): indicates that the URL specified as "user-portal-url" allows the user to extend a session once the client is no longer in a state of captivity. Thie provides a hint that a client system can suggest accessing the portal URL to the user when the session is near its limit in terms of time or bytes.
```
Comment (liucougar) [inline draft-ietf-capport-api.md]: typo: Thie

File: draft-ietf-capport-api.md [hunk 2]
```diff
@@ -122,7 +123,8 @@ Content-Type: application/captive+json
    "captive": true,
    "user-portal-url": "https://example.org/portal.html",
    "venue-info-url": "https://flight.example.com/entertainment",
-   "expire-date": "2014-01-01T23:28:56.782Z"
+   "expire-date": "2014-01-01T23:28:56.782Z",
```
Comment (liucougar) [inline draft-ietf-capport-api.md]: should this be changed to seconds-remaining?

**Review by liucougar — APPROVED** [review]

File: draft-ietf-capport-api.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-capport-api.md]: Thanks, fixed!

File: draft-ietf-capport-api.md (hunk 2 above)
Comment (tfpauly) [reply draft-ietf-capport-api.md]: Good catch, fixed!

**Review by mcr — APPROVED** [review]

File: draft-ietf-capport-api.md (hunk 1 above)
Comment (mcr) [inline draft-ietf-capport-api.md]: s/Thie/This/

**Comment by mcr** [comment]
I want to be clear that there is no extend-session API here. It's just an indication that the user could do that somehow, in a browser.

File: draft-ietf-capport-api.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-capport-api.md]: Yup, fixed =)

**Comment by tfpauly** [comment]
@mcr correct, there's no specific API to do so. It just indicates if there's a possibility of the user doing so. Future mechanisms for doing so automatically are left for the future =)

**Comment by danwing** [comment]
"Thie provides" -> "Thi**s** provides"

**Review by ekline — COMMENTED** [review]
LGTM

**Pull request merged**